README.md

```diff
@@ -32,7 +32,7 @@ In the coming months we will continue to refine the component implementations as
 See our [Getting Started Guide][getting-started]
 if you're building your first project with Angular-React.
 
-If you'd like to contribute, you must follow our [contributing guidelines](https://github.com/angular/material2/blob/master/CONTRIBUTING.md).
+If you'd like to contribute, you must follow our [contributing guidelines](hhttps://github.com/microsoft/angular-react/blob/master/CONTRIBUTING.md).
```

Comment:
Please fix this, invalid link.

Reply:
Whoops.  I originally copy pasted the url from the browser which had a capital M but noticed the README already had a link so I wanted to keep it consistent and missed highlighting the whole thing on the second past.  Fixed ⚡️